che-theia-activity-tracker/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "che-theia-activity-tracker",
```

Comment:
shouldn't it be `@eclipse-che/theia-activity-tracker` ?

Reply:
Yes, thanks, will fix